tools/src/test/java/com/nedap/archie/aom/TerminologyCodeConstraintsTest.java

```diff
@@ -71,10 +71,14 @@ public void externalTerminology() {
         CTerminologyCode code = new CTerminologyCode();
         code.setParent(new DummyRulesPrimitiveObjectParent(archetype));
         code.addConstraint("ac12");
-        //this is valid, because we have no idea about the binding currently. Could be invalid later, when we have proper
-        //external terminology support
+
+        ValidationConfiguration.setFailOnUnknownTerminologyId(false);
```

Comment:
The import for `ValidationConfiguration` is missing.

Reply:
Sorry about that, fixed!